rosdep/base.yaml

```diff
@@ -5887,7 +5887,9 @@ libqt5-core:
   opensuse: [libQt5Core5]
   rhel: [qt5-qtbase]
   slackware: [qt5]
-  ubuntu: [libqt5core5a]
+  ubuntu:
+    '*': [libqt5core5a]
+    noble: [libqt5core5t64]
```

Comment:
I think this will need to be the `'*'` and backfill older distros explicitly, in order to cover the upcoming releases. It's also this new name on oracular (24.10). @cottsay do you maybe have an opinion about this? If not we can bring it up at the next ROS 2 meeting.